.github/workflows/ci-backend.yml

```diff
@@ -328,69 +343,3 @@ jobs:
                   source .env.template
                   cd deploy
                   pytest multi_tenancy messaging -m "not skip_on_multitenancy"
-            - name: Run EE tests
-              env:
-                  PRIMARY_DB: 'clickhouse'
-              run: |
-                  cd deploy/
-                  pytest ee -m "not skip_on_multitenancy"
-                  pytest posthog -m "ee and not skip_on_multitenancy"
```

Comment:
Not sure about this, I think we may want to run the entire test suite within the Cloud scope, as there could be something that's overridden on Cloud that affects here.

Also, no tests on Cloud will have the `skip_on_multitenancy` tag so we can do, `pytest multi_tenancy messaging` as opposed to `pytest multi_tenancy messaging -m "not skip_on_multitenancy"`

Reply:
> as there could be something that's overridden on Cloud that affects here.

Is there? Can we mark these tests and suites specifically?

Reply:
We definitely could, but we're not doing that today (e.g. tests that override the signup serializer), we would have to update the relevant tests, and document this somehow to make sure we keep doing it. Though I'm a bit hesitant, will we remember to do this every time in practice?

Reply:
Not sure but I am sure we should try. 

My goal here is to arrive at a fast test suite and 1) running all tests on cloud and 2) the code around setup being different makes this hard. Rather than run the whole suite I think for cloud we should limit things to a specific suite.

Do you consider this blocking? I can look more deeply at this on wednesday.